django/utils/translation/trans_real.py

```diff
@@ -108,15 +108,21 @@ def __init__(self, language):
         self.__to_language = to_language(language)
         self.__locale = to_locale(language)
 
+        # Gets the list of additional domain names to search for.
+        # By default the only (and unremovable) one is 'django'.
+        domains = ['django']
+        if settings.LOCALE_FILENAMES:
+            domains += settings.LOCALE_FILENAMES
```

Comment:
Since `LOCALE_FILENAMES` defaults to an empty list I'd suggest you make `domains` unconditional append it to `['django']`.